Modules/_testexternalinspection.c

```diff
@@ -208,48 +225,49 @@ get_py_runtime_macos(pid_t pid)
             continue;
         }
 
+        if ((region_info.protection & VM_PROT_READ) == 0
+            || (region_info.protection & VM_PROT_EXECUTE) == 0) {
```

Comment:
```suggestion
        if (!(region_info.protection & (VM_PROT_READ | VM_PROT_EXECUTE))) {
```

Wouldn't this be sufficient?

Reply:
Yes, but it reads worse IMO. The former is pretty obvious: "if VM_PROT_READ is unset or VM_PROT_EXECUTE is unset". The latter reads: "if not both VM_PROT_READ and VM_PROT_EXECUTE are set". The negative there and the compound bit op makes it hard to understand what's going on.